packages/enzyme-adapter-utils/src/elementToTree.js

```diff
@@ -0,0 +1,65 @@
+const ITERATOR_SYMBOL = typeof Symbol === 'function' && Symbol.iterator;
```

Comment:
I saw that enzyme has this elsewhere, but in a different package. If there's a better way to share this across lerna packages, please let me know 😄